sn_networking/src/error.rs

```diff
@@ -78,4 +78,7 @@ pub enum Error {
 
     #[error("Record was not found locally")]
     RecordNotFound,
+
+    #[error("Bincode error:: {0}")]
+    Bincode(#[from] bincode::Error),
```

Comment:
I think we'll want to reframe this error for the lib. What's relevant, that bincode errored or that what we put into bindcode was wrong?